fraud-detection-update-sample/fraud_detection_fetch_land_use_data.py

```diff
@@ -1,8 +1,10 @@
 import ee
 
-MAPBIOMAS_BRAZIL_ANNUAL_WATER_COVERAGE_IMAGE = 'projects/mapbiomas-workspace/public/collection6/mapbiomas_water_collection1_annual_water_coverage_v2'
+MAPBIOMAS_BRAZIL_ANNUAL_WATER_COVERAGE_GEE_IMAGE = 'projects/mapbiomas-workspace/public/collection8/mapbiomas_water_collection2_annual_water_coverage_v1'
```

Comment:
"collection8" seems pretty random.
this is a "public/collection" so we should complain to them about breaks and see if we can get them to stabilize.